autoload/copilot.vim

```diff
@@ -430,7 +429,7 @@ function! s:UpdatePreview() abort
     else
       let data.virt_text += annot
     endif
-    call nvim_buf_del_extmark(0, copilot#NvimNs(), 1)
+    let data.hl_mode = 'combine'
```

Comment:
no need to call `nvim_buf_del_extmark` -- `nvim_buf_set_extmark` will overrides the old one

Reply:
This works around a bug in some versions of Neovim.